Makefile

```diff
@@ -86,14 +85,27 @@ stability-test-push: stability-test-docker
 fault-trigger:
 	$(GO) -ldflags '$(LDFLAGS)' -o tests/images/fault-trigger/bin/fault-trigger tests/cmd/fault-trigger/*.go
 
+# ARGS:
+#
+# GOFLAGS: Extra flags to pass to 'go' when building, e.g.
+# 		`-v` for verbose logging.
+# 		`-race` for race detector.
+# GO_COVER: Whether to run tests with code coverage. Set to 'y' to enable coverage collection.
+#
+ifeq ($(GO_COVER),y)
```

Comment:
I disable code coverage by default because it's not needed in local development. This will speed up local unit tests.

After this is accepted and merged, I'll update our master job to enable coverage.